docs/k0s-single-node.md

```diff
@@ -33,21 +20,13 @@ k0s default-config | tee ${HOME}/.k0s/k0s.yaml
 
```

Comment:
We have now two different methods on creating k0s.yaml:
`k0s default-config | tee ${HOME}/.k0s/k0s.yaml` and `k0s default-config > k0s.yaml` let's decide on one, and stick to it.